docs/samplers.txt

```diff
@@ -72,6 +70,17 @@ MCMC samplers
 - pymc :code:`bilby.core.sampler.pymc.Pymc`
 - zeus :code:`bilby.core.sampler.zeus.Zeus`
 
+-------------------
+Deprecated samplers
+-------------------
+
+For some samplers that are not actively maintained, the maintenance burden
+outweighed the benefit of shipping the interfaces with :code:`bilby`.
+The following samplers are no longer actively supported in the main package,
+but are available via plugin packages.
+
+- PyPolyChord :code:`pypolychord-bilby`
+- DNest4 :code:`dnest4-bilby`
```

Comment:
I think should be slightly reworded since some samplers (just nessai for now) have support via plugins and that is the preferred way to use the sampler.